src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -50,11 +46,20 @@ public CommandResult execute() {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            final Set<String> upperCasedWordsInName = toLowerSet(wordsInName);
+            if (!Collections.disjoint(upperCasedWordsInName, keywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+    private Set<String> toLowerSet(Set<String> set){
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.